### PR TITLE
KapEmoji

### PR DESCRIPTION
**Pull request by andertun**
Add KapEmoji component and the two first emojis

`<KapEmoji.ThumbsUp />`
`<KapEmoji.Wave />`

image

### CODE REVIEW

**Comment by andertun** [comment]
npm publish

File: src/emojis/EmojiWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        width = sizeInPx;
+        height = sizeInPx / ratio;
+    } else {
+        width = sizeInPx * ratio;
+        height = sizeInPx;
+    }
+
+    // Ensure the dimensions do not exceed the max size
+    width = Math.min(width, sizeInPx);
+    height = Math.min(height, sizeInPx);
```
Comment (wejendorp) [inline src/emojis/EmojiWrapper.tsx:45]: this doesn't seem right, it ignores the original ratio. Don't we risk it getting squashed?

Comment (andertun) [reply src/emojis/EmojiWrapper.tsx]: It uses the ratio in the if-else above. But I pushed something new that should be better

**Review by wejendorp — APPROVED** [review]
Awesomesauce! 👍

File: src/emojis/EmojiWrapper.tsx (hunk 1 above)
Comment (wejendorp) [reply src/emojis/EmojiWrapper.tsx]: Much more readable now, thanks

**Pull request merged**